crates/oxc_linter/src/snapshots/no_focused_tests.snap

```diff
@@ -126,3 +126,46 @@ source: crates/oxc_linter/src/tester.rs
    · ─────────────────
    ╰────
   help: Remove focus from test.
+
+  ⚠ eslint-plugin-vitest(no-focused-tests): Focused tests are not allowed.
+   ╭─[no_focused_tests.tsx:3:13]
+ 2 │             import { it } from 'vitest'; 
+ 3 │             it.only("test", () => {});
```

Comment:
Should we change the label to just highlight `only`?

Reply:
Done.